src/components/size/size.scss

```diff
@@ -0,0 +1,133 @@
+@import '../../assets/scss/variables';
+@import '../../assets/scss/utilities';
+
+:root {
+  --element-spacing-lg: var(--space-lg-width-gutter);
+  --element-spacing-sm: var(--space-sm-width-gutter);
+}
+
```

Comment:
Instead of `:root` We might want to add a base `element--alignment` and `element--size` classes here to define the CSS variables on for each element.

Reply:
We would be defining the variables for size and alignment though right? we wouldn't be re-defining variables per usage? Sorry, just trying to understand.

Reply:
ex:

```
*[class*="element--alignment"], *[*[class*="element--size"] {  

    --element-spacing-lg: var(--space-lg-width-gutter);  
    --element-spacing-sm: var(--space-sm-width-gutter);
}
```

Reply:
I think @pyrello mentioned that we might want to move these to an `inline.scss` namespace?    I'm thinking about locally scoping the variables for just this component as discussed here https://blog.hubspot.com/website/css-variables.    We are using the global variables but setting their namespace at the component level `--element-spacing-sm: var(--space-sm-width-gutter);`.

Reply:
@GaryRidgway Yes, that's what I'm thinking but ideally would target these through BEM like `element-inline element-inline--size` and `element-inline element-inline--alignment` or something like that.